core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -923,15 +971,13 @@ class SparkContext(config: SparkConf) extends Logging {
   /**
    * Load data from a flat binary file, assuming the length of each record is constant.
    *
-   * @note We ensure that the byte array for each record in the resulting RDD
+   * @note we ensure that the byte array for each record in the resulting RDD
```

Comment:
I am not sure of making this lower-cased because we will see this as the start of sentence 
- scaladoc
  ![2016-12-06 1 02 12](https://cloud.githubusercontent.com/assets/6477701/20912647/38ccf9e4-bbb4-11e6-8346-42cd6297d075.png)

- javadoc
  ![2016-12-06 1 02 05](https://cloud.githubusercontent.com/assets/6477701/20912648/38d2160e-bbb4-11e6-8d72-1c4480ca2276.png)